wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -288,12 +287,16 @@ class KotlinGenerator private constructor(
     type.fieldsWithJavaInteropOneOfs().forEach { field ->
       val fieldName = nameAllocator[field]
 
+      val propertyBuilder = PropertySpec.builder(fieldName, field.declarationClass)
+          .mutable(true)
```

Comment:
nit: `true` can be omitted

Reply:
`.mutable(true)` was there before; i kinda like the explicitness